pkg/ddc/alluxio/transform.go

```diff
@@ -127,7 +129,10 @@ func (e *AlluxioEngine) transformCommonPart(runtime *datav1alpha1.AlluxioRuntime
 	imageTag := runtime.Spec.AlluxioVersion.ImageTag
 	imagePullPolicy := runtime.Spec.AlluxioVersion.ImagePullPolicy
 
-	value.Image, value.ImageTag, value.ImagePullPolicy = e.parseRuntimeImage(image, imageTag, imagePullPolicy)
+	// TODO: support imagePullSecrets by AlluxioRuntime
+	imagePullSecrets := []corev1.LocalObjectReference{}
+
+	value.Image, value.ImageTag, value.ImagePullPolicy, value.ImagePullSecrets = e.parseRuntimeImage(image, imageTag, imagePullPolicy, imagePullSecrets)
```

Comment:
Why should we pass an empty `imagePullSecrets` object into the function?

Reply:
> Why should we pass an empty `imagePullSecrets` object into the function?

Prepare for the subsequent support of pass image pull secret directly through Alluxioruntime